test/system/dates_helper_test.rb

```diff
@@ -43,20 +43,12 @@ class DatesHelperTest < ApplicationSystemTestCase
         click_on "Back"
 
         # Assert today's date is displayed correctly.
-        assert page.has_text? "Today at #{time.strftime("%l:%M %p").strip}"
+        assert page.has_text? I18n.l(time, format: :default)
```

Comment:
It looks like this assertion is failing on CI, but the error message isn't that descriptive, I wonder if we should use Capybara `assert_text` method instead?

```suggestion
        assert_text I18n.l(time, format: :default)
```